include/RAJA/policy/sycl/kernel/SyclKernel.hpp

```diff
@@ -186,7 +161,7 @@ struct SyclLaunchHelperNonTrivial<sycl_launch<async0>,StmtList,Data,Types>
  * determined at runtime using the SYCL occupancy calculator.
  */
 template<bool async0, typename StmtList, typename Data, typename Types>
-struct SyclLaunchHelper<sycl_launch<async0>,StmtList,Data,Types>
+struct SyclLaunchHelper<true,sycl_launch<async0>,StmtList,Data,Types>
```

Comment:
This launcher will never `wait()` on line 189 below?

Reply:
Nevermind, this `true` is referring to the `bool` on line 111, which I believe is representing trivially copyable?

Reply:
Yes.  We are doing this partial specialization to dispatch to either the non_trivial or trivial version of the forall.